examples/bzlmod/BUILD.bazel

```diff
@@ -87,3 +87,38 @@ build_test(
     name = "all_requirements",
     targets = all_requirements,
 )
+
+exports_files(
+    glob(["data/**"]),
+    visibility = ["//visibility:public"],
+)
+
+#py_test(
```

Comment:
@rickeylev I'm getting an error that the test cannot find the dependency "@rules_python//python/runfiles".  Any ideas?

Reply:
bzlmod slack helped me out